assets/i18n/ast.json

```diff
@@ -5,7 +5,7 @@
 	"I18N_ABOUT_PAGE_ABOUT_TAB_HEADING": "Tocante a Oppia",
 	"I18N_ABOUT_PAGE_ABOUT_TAB_SHARE": "Publicar y compartir",
 	"I18N_ABOUT_PAGE_ABOUT_TAB_SHARE_TEXT": "les tos creaciones cola comunidá.",
-	"I18N_ABOUT_PAGE_CREDITS_TAB_HEADING": "Créditos",
+	"I18N_CREDITS_PAGE_HEADING": "Créditos",
```

Comment:
This needs to be alphabetized correctly, otherwise tests will fail. Ditto elsewhere.